### DISCUSSION TITLE
03.07.00 & DNN 9.11

### DISCUSSION THREAD

**Discussion by renderfarmer** [HotcakesCommerce/hotcakes-commerce-core#461] [Q&A]
Does version [03.07.00 work with latest DNN 9.11 ??

**Comment by Orrouk** [comment]
Haven't try yet

**Reply by james7342** [reply]
I have it installed on DNN 9.11.2.

**Comment by WillStrohl** [comment]
We haven't found anything yet to tell us it's not compatible, nor have we had any customer/community reports toward the same end.  It should work fine.  If not, please create an **detailed** issue so we can resolve any issues ASAP.  Thanks!
